app/src/main/res/layout/activity_bottom_navigation.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This should be a LinearLayout with the viewpager with a weight of 1 as height, seems easier to read (perf-wise it's the same)